examples/control_character/replay.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import os
+
+import numpy as np
+import pandas as pd
+import spear
+
+
+def get_action(row):
```

Comment:
This function is poorly named. Why is the variable being returned called `bone_transforms` and the function is called `get_action`? It's also hard to read.

```
def get_transforms(transform_names, df):
    transforms = {}
    for transform_name in transform_names:
        transform = {
            "Location": {"X": df[transform_name + "_location_x"],       "Y": df[transform_name + "_location_y"],         "Z": df[transform_name + "_location_z"]},
            "Rotation": {"Roll": df[transform_name + "_rotation_roll"], "Pitch": df[transform_name + "_rotation_pitch"], "Yaw": df[transform_name + "_rotation_yaw"]},
            "Scale3D": {"X": df[transform_name + "_location_x"],        "Y": df[transform_name + "_location_y"],         "Z": df[transform_name + "_location_z"]}}
        transforms[transform_name] = transform
```